src/libraries/Microsoft.AspNetCore.Internal.Transport/src/Microsoft.AspNetCore.Internal.Transport.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.Build.NoTargets">
```

Comment:
Shouldn't this file be a .proj and not a .csproj as it doesn't contain any C# source files?